bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/java/org/openhab/binding/bluetooth/daikinmadoka/internal/model/commands/GetVersionCommand.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.bluetooth.daikinmadoka.internal.model.commands;
+
+import java.util.concurrent.Executor;
+
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.bluetooth.daikinmadoka.internal.model.MadokaMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * @author blafois
+ *
+ */
+public class GetVersionCommand extends BRC1HCommand {
+
+    private final Logger logger = LoggerFactory.getLogger(GetVersionCommand.class);
+    private String remoteControllerVersion;
+    private String communicationControllerVersion;
+
+    @Override
+    public byte[] getRequest() {
+        return MadokaMessage.createRequest(this);
+    }
+
+    @Override
+    public boolean handleResponse(Executor executor, ResponseListener listener, byte @Nullable [] response) {
+        if (response == null) {
+            return false;
+        }
+
+        try {
+
+            MadokaMessage mm = MadokaMessage.parse(response);
+
+            int remoteControllerMajor = Integer.valueOf(mm.getValues().get(0x45).getRawValue()[0]);
```

Comment:
Please cache `mm.getValues().get(0x45).getRawValue()` and `mm.getValues().get(0x46).getRawValue()` in local variables so you can reuse them.